opentracing-api/src/main/java/io/opentracing/ActiveSpan.java

```diff
@@ -106,4 +142,13 @@
         ActiveSpan activate();
     }
 
+    /**
+     * The {@link ActiveSpan.Observer} interface learns of capture, activate, and deactivate calls for a given
+     * {@link ActiveSpan} instance and any of its "descendants" in the capture/activate tree.
+     */
+    interface Observer {
+        void afterCapture(ActiveSpan captured, Continuation destination);
```

Comment:
These methods look a bit fancy. Isn't it better to just pass the span in all of them?